### ISSUE TITLE
modify profile to user "default" will change password to "default" and cause other problems

### ISSUE THREAD

**Issue by xiaozzz** [Altinity/clickhouse-operator#282]
Hi,
I want to change the permission of default user to forbid ddl, so I have the following config:
```yml
profiles:
  not_allow_ddl/allow_ddl: 0
users:
  default/profile: not_allow_ddl
  admin/password: admin
  admin/profile: default
```
but these codes change the password to "default" like this
https://github.com/Altinity/clickhouse-operator/blob/master/pkg/apis/clickhouse.altinity.com/v1/type_config.go#L267-L269 
```yml
users.xml: |
<yandex>
    <users>
        <admin>
            <networks>
                <ip>::/0</ip>
            </networks>
            <password>admin</password>
            <profile>default</profile>
            <quota>default</quota>
        </admin>
        <default>
            <networks>
                <ip>::/0</ip>
            </networks>
            <password>default</password>
            <profile>not_allow_ddl</profile>
            <quota>default</quota>
        </default>
```
I am working on a shard env, then I have found this issue https://github.com/Altinity/clickhouse-operator/issues/265 which means I should not change the password, and I do not want to change it either

please help me.

**Comment by alex-zaitsev** [comment]
@xiaozzz 
As a workaround, you can specify empty password for default user explicitly.

We will fix this in an upcoming hot fix release later today or tomorrow

**Comment by xiaozzz** [comment]
@alex-zaitsev 
I have tried leave the password blank like this
```yml
users:
      default/password:
```
or
```yml
users:
      default/password: ~
```
but the config map translate it to newline like this
```yml
<default>
    <networks>
        <ip>::/0</ip>
    </networks>
    <password>
    </password>
    <profile>not_allow_ddl</profile>
    <quota>default</quota>
</default>
```
and clickhouse still ask me for password.

Thank you for your quick apply, and i will wait for your hotfix

**Comment by alex-zaitsev** [comment]
Fixed in 0.9.3
